src/cascadia/TerminalSettingsModel/TerminalSettings.cpp

```diff
@@ -118,6 +118,14 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             {
                 settings.SuppressApplicationTitle(newTerminalArgs.SuppressApplicationTitle().Value());
             }
+            if (!newTerminalArgs.ColorScheme().empty())
+            {
+                const auto schemes = appSettings.GlobalSettings().ColorSchemes();
+                if (const auto scheme = schemes.TryLookup(newTerminalArgs.ColorScheme()))

```

Comment:
```suggestion
                if (const auto& scheme = schemes.TryLookup(newTerminalArgs.ColorScheme()))
```

Who knows, the compiler may optimize this one. Probably won't optimize the above one.